README_uwp.md

```diff
@@ -0,0 +1,183 @@
+# Python for UWP
+
+PythonUWP is based on a modified version of CPython 3.5 which supports running as a Windows 10 UWP (Universal Windows Platform) application.  Specifically, this supports running in a background application on Windows 10 IoT core.
```

Comment:
I would link "UWP" with https://msdn.microsoft.com/en-us/library/windows/apps/dn894631.aspx. Just for people who want to look into what it means in more detail.